src/components/header/Header.tsx

```diff
@@ -44,6 +45,8 @@ const GithubLink = styled(Link)`
 `;
 
 const Header = () => {
+  const isMobile = useMediaQuery('(max-width: 740px)');
```

Comment:
I've also thought about managing multiple iterations of isMobile in one place, but I don't think it makes much sense!

If you need to use useMediaQuery more extensively

`const MOBILE_LAYOUT = 'max-width: 740px'`
I think you can use it like that!
But I think it's highly likely not 😻🚀